stingray/crossspectrum.py

```diff
@@ -1,11 +1,12 @@
-from __future__ import division, absolute_import, print_function
+
```

Comment:
Why have you deleted __future__ imports? :)
I think that print_function and division should definitely be there